<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPEFileCreatorForm</class>
 <widget class="QWidget" name="HPEFileCreatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileCreator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	padding: 2px 8px;
     color: gray;
     background-color: white;
     border-style: solid;
     border-width: 5px;
     border-radius: 8px;
     border-color: rgb(208, 208, 208);
     font: bold 15px;
}

QPushButton:hover
{
	/*border-color: gray;*/
     color: rgb(255, 255, 255);
     background-color: rgb(208, 208, 208);
}

QPushButton:disabled
{
	color: rgb(235, 235, 235);
}

QComboBox {
	color: gray;
	padding-left: 8px;
	border: 4px solid gray;
	border-radius: 8px;
	selection-background-color: rgb(235, 235, 235);
}

QComboBox  QPushButton:hover {
	
	background-color: rgb(235, 235, 235);
}

QComboBox::drop-down:button {
	background: white;
	border-radius: 8px;
}

QComboBox::down-arrow {
	width: 12px;
	height: 12px;
	image: url(:/icons/resources/icons/ico_arrow_down.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="backButtonContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentDirLabel">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>Current Dir</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="titleEditContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>36</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>36</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QComboBox" name="layoutSelector">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>4</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleEdit">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
	padding: 0 5px;
	color: rgb(235, 235, 235);
	border-bottom: 2px solid rgb(235, 235, 235);
}

QLineEdit::focus
{
	color: gray;
	border-bottom: 2px solid gray;
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="frontMatterEdit">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="confirmButtonContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
